sphinx_js/templates/attribute.rst

```diff
@@ -15,3 +22,11 @@
    {%- endif %}
 
    {{ content|indent(3) }}
+
+   {% if seealso -%}
+   .. seealso::
+
+      {% for reference in seealso -%}
+      - :any:`{{ reference }}`
```

Comment:
Does this work in the js domain? I [see it documented](http://www.sphinx-doc.org/en/stable/domains.html) for C++ but nowhere else.

Reply:
It seems to be [new to Sphinx 1.3](http://www.sphinx-doc.org/en/stable/markup/inline.html#role-any), and should work with any domain. Do we want to be compatible with earlier versions of Sphinx though?

Reply:
No, I'm not concerned with it.